lace/lace_cc/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "lace_cc"
-version = "0.1.3"
+version = "0.1.4"
```

Comment:
More version talk--bumping the local dependencies probably means bumping this version makes sense. Otherwise you would have a version conflict when building the local stuff.